File: keyboards/reply.py
from aiogram.types import ReplyKeyboardMarkup, KeyboardButton
from utils.texts import TEXTS

def get_lang_keyboard() -> ReplyKeyboardMarkup:
    """Til tanlash tugmalari"""
    return ReplyKeyboardMarkup(
        keyboard=[
            [
                KeyboardButton(text="🇺🇿 O‘zbek tili"),
                KeyboardButton(text="🇷🇺 Rus tili"),
                KeyboardButton(text="🇬🇧 Ingliz tili")
            ]
        ],
        resize_keyboard=True,
        input_field_placeholder="Tilni tanlang / Выберите язык / Select language"
    )

def get_main_menu(lang: str) -> ReplyKeyboardMarkup:
    """Asosiy menyu tugmalari"""
    texts = TEXTS.get(lang, TEXTS['uz'])
    
    return ReplyKeyboardMarkup(
        keyboard=[
            [KeyboardButton(text=texts['btn_tender_uz'])],
            [KeyboardButton(text=texts['btn_exh_uz'])],
            [KeyboardButton(text=texts['btn_exh_glb'])],
            [KeyboardButton(text=texts['btn_products'])]
        ],
        resize_keyboard=True,
        input_field_placeholder=texts['main_menu']
    )

File: cf_bot/js_bot/keyboards/reply.js
const { Markup } = require('telegraf');
const { TEXTS } = require('../utils/texts');

/**
 * Til tanlash tugmalari
 */
function getLangKeyboard() {
    return Markup.keyboard([
        ['🇺🇿 O‘zbek tili', '🇷🇺 Rus tili', '🇬🇧 Ingliz tili']
    ])
    .resize()
    .oneTime(false)
    .placeholder("Tilni tanlang / Выберите язык / Select language");
}

/**
 * Asosiy menyu tugmalari
 * @param {string} lang 
 */
function getMainMenu(lang) {
    const texts = TEXTS[lang] || TEXTS['uz'];
    
    return Markup.keyboard([
        [texts.btn_tender_uz],
        [texts.btn_exh_uz],
        [texts.btn_exh_glb],
        [texts.btn_products]
    ])
    .resize()
    .oneTime(false)
    .placeholder(texts.main_menu);
}

module.exports = {
    getLangKeyboard,
    getMainMenu
};
